hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/extensions/SASTokenProvider.java

```diff
@@ -51,13 +50,6 @@ public interface SASTokenProvider {
   String SET_PROPERTIES_OPERATION = "set-properties";
   String WRITE_OPERATION = "write";
 
-
-  // Filesystem Level Operations
```

Comment:
Is the purpose for removing these related to SAS issue for container API's ?

Reply:
Yes, that's right.
Theya re not there anyway on trunk. They were added in this PR only and removed later.